features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/tracking/ActivityLifecycleTrackingStrategy.kt

```diff
@@ -89,7 +91,17 @@ abstract class ActivityLifecycleTrackingStrategy :
 
     @MainThread
     override fun onActivityCreated(activity: Activity, savedInstanceState: Bundle?) {
-        // No Op
+        val intent = activity.intent
+        val extras = intent.extras
+        val testId = extras?.getString("_dd.synthetics.test_id")
+        val resultId = extras?.getString("_dd.synthetics.result_id")
+        if (!testId.isNullOrBlank()) {
```

Comment:
why do we check only `testId` here, if later on in the attributes injection logic we check both `testId` and `resultId` to be able to add them? Would it make more more sense for the `SetSyntheticsTestAttribute` to have both properties as non-null then?